Index: neovim-0.6.1/src/nvim/terminal.c
===================================================================
--- neovim-0.6.1.orig/src/nvim/terminal.c	2021-12-31 18:02:21.000000000 +0100
+++ neovim-0.6.1/src/nvim/terminal.c	2022-01-01 07:16:58.142880441 +0100
@@ -844,7 +844,11 @@ static int term_settermprop(VTermProp pr
 
   case VTERM_PROP_TITLE: {
     buf_T *buf = handle_get_buffer(term->buf_handle);
+#if VTERM_VERSION_MAJOR > 0 || (VTERM_VERSION_MAJOR == 0 && VTERM_VERSION_MINOR >= 2)
+    buf_set_term_title(buf, val->string.str);
+#else
     buf_set_term_title(buf, val->string);
+#endif
     break;
   }
 
